### PR TITLE
Japanese version

### LINKED ISSUE #37: Languages / localization: Convert Japanese site to new format and translate basic content.
Create subdomain (jp.wgtwo.com) or a structure like wgtwo.com/jp. Need to prepare our platform for multiple languages in the future, so we need to make it scaleable. Yet, important that we can make the localized pages relatively static to avoid continuous updating.

### PR DESCRIPTION
**Pull request by s-utstrand**
Prepare site for multi-language and add Japanese translation.

Closes #37

### CODE REVIEW

File: website/src/pages/index.tsx [hunk 1]
```diff
@@ -24,7 +27,7 @@ function Index() {
         <Enterprise />
         <Quote />
         <Integrations />
-        <NewsletterSignup />
+        <NewsletterSignup hidden={i18n.currentLocale !== "en"} />
```
Comment (jorunfa) [inline website/src/pages/index.tsx:30]: @torotimes Do we want to hide newsletter signup for non-english speakers?

File: website/i18n/jp/code.json [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+  "home.rbe.api.description": {
+    "message": "wgtwo APIを利用して加入者のプロビジョニングを柔軟に実施",
+    "description": "Description for the API deployment model"
+  },
+  "home.rbe.telco.description": {
+    "message": "セキュアで、開発しやすい、次世代の、そしてサービス豊富な通信プラットフォームで、お客様のビジネスを成功に導きます。End-of-Lifeは過去のものです。",
+    "description": "Description for the telco deployment model"
+  },
+  "theme.ErrorPageContent.title": {
+    "message": "This page crashed.",
```
Comment (jorunfa) [inline website/i18n/jp/code.json:91]: Do we need to add non-translated entries? Will it not fallback to english (default) anyways when it's missing?

File: website/src/components/casestudy/casestudy-list.tsx [hunk 3]
```diff
@@ -1,37 +1,94 @@
...
 
 const caseStudies = [
   {
     name: "Adjacent Cores for Operators",
-    description:
-      "Mobile Network Operators face the challenge of scalability on a daily basis. Highly complex networks that limit the number of use-cases and additional complexity they can take on is the norm.",
+    description: (
+      <Translate
+        id="home.casestudy.adjacentcores.description"
+        description="Description for the adjacent cores for operators case study"
```
Comment (jorunfa) [inline website/src/components/casestudy/casestudy-list.tsx:12]: The description is supposed to be repeated both here and in the translation (i.e. website/i18n/jp/code.json)?

Comment (s-utstrand) [reply website/src/components/casestudy/casestudy-list.tsx:12]: Yes. When you have wrapped the text that should be translatable in the `<Translate>` component, you run `docusaurus write-translations` and the translation files (`/i18n/jp/**/*.json`) are automatically generated. The description are there in the json file for you to understand what the texts refer to.

File: website/src/pages/index.tsx (hunk 1 above)
Comment (s-utstrand) [reply website/src/pages/index.tsx:30]: I couldn't find any reference right now, so I might have misunderstood or remember wrong. But I think it was mentioned on several occasions that the newsletter form should only be on the English site. Correct me if I'm wrong, @torotimes

**Review by jorunfa — APPROVED** [review]

File: website/src/pages/index.tsx (hunk 1 above)
Comment (torotimes) [reply website/src/pages/index.tsx:30]: Correct.

**Pull request merged**